conformance/tests/generics_syntax_infer_variance.py

```diff
@@ -2,27 +2,30 @@
 Tests the handling of "infer_variance" parameter for TypeVar.
 """
 
+from dataclasses import dataclass
+
 # Specification: https://peps.python.org/pep-0695/#auto-variance-for-typevar
 
 from typing import Final, Generic, Iterator, Sequence, TypeVar
-from dataclasses import dataclass
 
 
 T = TypeVar("T", infer_variance=True)
 K = TypeVar("K", infer_variance=True)
 V = TypeVar("V", infer_variance=True)
 
-S1 = TypeVar("S1", covariant=True, infer_variance=True)  # E: cannot use covariant with infer_variance
+S1 = TypeVar(
+    "S1", covariant=True, infer_variance=True
+)  # E: cannot use covariant with infer_variance
```

Comment:
It looks like you reformatted this line and the one below. This changes the test because the comment is now on the wrong line.